Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.cpp

```diff
@@ -1701,20 +1721,40 @@ namespace AZ
         void DirectionalLightFeatureProcessor::SetFullscreenPassSettings()
         {
             ShadowProperty& shadowProperty = m_shadowProperties.GetData(m_shadowingLightHandle.GetIndex());
+            bool fullscreenBlurEnabled = shadowProperty.m_fullscreenBlurEnabled;
 
+#if USING_SCREENSHADOW_COMPUTEPASS
+            if (m_fullscreenShadowComputePass)
+            {
+                m_fullscreenShadowComputePass->SetEnabled(fullscreenBlurEnabled);
```

Comment:
Do we not want full screen shadow pass without blur? Trying to understand what would happen if we disable the blur. Will shadows stop working?

Reply:
Because if we don't need the full-screen shadow to blur, the full-screen shadow pass has some performance cost. You can find it in DirectionalLightShadow.azsli. If we don't use full-screen blur, then directly calculating shadow will be used.